client/pool.c

```diff
@@ -34,10 +34,10 @@
 #define START_MINERS_COUNT     256
 #define START_MINERS_IP_COUNT  8
 
-#define HEADER_WORD                        0x3fca9e2bu
 #define FUND_ADDRESS                       "FQglVQtb60vQv2DOWEUL7yh3smtj7g1s" /* community fund */
 #define SHARES_PER_TASK_LIMIT              20                                 /* maximum count of shares per task */
 #define DEFAUL_CONNECTIONS_PER_MINER_LIMIT 100
+#define WORKERNAME_HEADER_WORD             0xf46b9853u
```

Comment:
How do you define this value? Random or have some explanation on this value?

Reply:
Just random value.